nncf/api/compression.py

```diff
@@ -31,6 +31,9 @@ class CompressionLoss(ABC):
     sparsity algorithm calculates the number of non-zero weights in convolutional
     and fully-connected layers to construct the loss function.
     """
+    def __init__(self):
+        super().__init__()
+        self.current_sparsity = 0.0
```

Comment:
I understand you need to fix this error:
`nncf/common/sparsity/schedulers.py:273: error: "CompressionLoss" has no attribute "current_sparsity"  [attr-defined]`
but this is not the right way to do it since this exact class is a) a pure interface, and is not supposed to have `__init__`, and b) even if `self.current_sparsity` were made a property, `CompressionLoss` would not be the right place for it since the `current_sparsity` is only relevant to sparsity-related loss use cases and not for the general ABC such as `CompressionLoss`.

So this change most definitely cannot be accepted, but we need to think how to correct the situation. Analyzing the original problem at nncf/common/sparsity/schedulers.py:273 it now seems somewhat strange to me that the current sparsity value should really be accessible through the `SparsityController.scheduler.loss` field and not, say, through the `SparsityController` itself. Another solution would be to make sure that the `SparsityController` class definition types the `loss` field/property to have the `current_sparsity` attribute, i.e. by typing the `loss` to be something like `SparsityLoss`. 

@andrey-churkin what do you think?